docs/superpowers/specs/2026-04-23-theme-color-toggle-design.md

```
@@ -0,0 +1,213 @@
+**`client/package.json`**
+
+- Add the `sight.depthColors.enabled` contribution under
+  `contributes.configuration.properties`.
+
+**`README.md`** (brief — one or two sentences where depth colors are
+described): note that the feature can be turned off via
+`sight.depthColors.enabled`.
+
+### Data flow
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)